### PR TITLE
fix: fix back

### PR DESCRIPTION
**Pull request by Doroshkeviches**

### CODE REVIEW

File: frontend/src/Pages/nutrityionix_CalculatorPage/style/index.sass [hunk 1]
```diff
@@ -134,6 +143,51 @@
...
+		padding: 10px 10px
+
+@media screen and (max-width: 480px)
+	.photo 
+		margin-left: 0px
+		&_image 
+			max-width: 60px
+			max-height: 60px	
+		&_container 
+			width: 60px
```
Comment (inanimate631) [inline frontend/src/Pages/nutrityionix_CalculatorPage/style/index.sass:193]: new line and mb create new file for adaptative

**Review by inanimate631 — APPROVED** [review]

File: backend/authRouter.js [hunk 2]
```diff
@@ -24,5 +24,6 @@ router.put('/updateUser', controller.changeValue);
 router.get('/exercisesList', controller.createExerciseList);
 router.get('/exercises/:name', controller.createExercise);
 router.get('/workout/:name', controller.createWockout);
+router.put('/createMyWorkout',controller.createMyWorkout )
```
Comment (RetiredHedgehog) [inline backend/authRouter.js:27]: Why do you use `put` for `create` operation? It's better to use `post` , I think.

**Review by RetiredHedgehog — APPROVED** [review]

File: backend/authController.js [hunk 3]
```diff
@@ -191,5 +191,17 @@ class authController {
     );
     res.json(resultArray);
   }
+  async createMyWorkout(req,res){
+    try {
+      const {username, workout} = req.body
+      const user = await User.findOne({ username })
+      console.log(typeof workout)
```
Comment (JaneEllison) [inline backend/authController.js:198]: Do you need this console log?

File: frontend/src/Pages/nutrityionix_CalculatorPage/style/index.sass [hunk 4]
```diff
@@ -134,6 +143,51 @@
...
+#total_container_item_nutritions_fat,
+#total_container_item_nutritions_carbohydrate
+	display: flex
+	flex-direction: column
+	align-items: center
+
 	
+@media screen and (max-width: 1024px)
+	#calculateNutritionContainer
+		font-size: 14px
```
Comment (JaneEllison) [inline frontend/src/Pages/nutrityionix_CalculatorPage/style/index.sass:159]: variable?

File: frontend/src/Pages/nutrityionix_CalculatorPage/style/index.sass [hunk 5]
```diff
@@ -134,6 +143,51 @@
...
+
+@media screen and (max-width: 768px)
+	#calculateNutritionContainer
+		font-size: 14px
```
Comment (JaneEllison) [inline frontend/src/Pages/nutrityionix_CalculatorPage/style/index.sass:163]: variable?

**Review by JaneEllison — APPROVED** [review]

**Pull request merged**